source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -44,11 +44,41 @@ matching data is inserted.
 Find
 ----
 
-The ``find()`` method is called on the ``Collection`` object that
-references the collection you want to query. The method accepts a query
-document that describes the documents you want to retrieve. For more
-information on how to specify your query document, see our guide on
-how to :doc:`Specify a Query </fundamentals/crud/query-document>`.
+You can call the ``find()`` method on a ``Collection`` object. The
+method accepts a query document that describes the documents you want to
+retrieve. For more information on how to specify your query document,
+see our guide on how to :ref:`node-fundamentals-query-document`.
+
+.. tip:: No Query Criteria
+   
+   To execute a find operation that has no query criteria, you can
+   pass an empty query or omit the query document in your find
+   method parameters.
+
+   The following operations both return all documents in the
+   ``myColl`` collection:
+
+   .. code-block:: javascript
+
+      await myColl.find(); // no query
+      await myColl.find({}); // empty query
+   
+   If you don't pass a query or pass an empty query
+   to the ``findOne()`` method, the operation returns the first
+   document of a collection.
+
+   Even if you pass an empty query in a find operation, you can
+   specify options. For example, the following code shows how you can
```

Comment:
[suggestion]

I'd recommend switching this to put the imperative of what readers can do before the conditional.

```suggestion
   You can
   specify options in a find operation even when you pass an empty query. For example, the following code shows how you can
```